PSController/PSController/Controller/ViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ViewController.swift
+//  PSController
+//
+//  Created by Burak Sak on 1.05.2021.
+//
+
+import UIKit
+
+final class ViewController: UIViewController {
+    
+    @IBOutlet private weak var collectionView: UICollectionView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var basketButton: UIButton!
+    @IBOutlet private weak var controllerTabView: UIView!
+    @IBOutlet private weak var devicesTabView: UIView!
+    @IBOutlet private weak var mouseTabView: UIView!
+    
+    private var cellWidth: CGFloat = 0
+    private let products: [Product] = [.init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "blackPSController")),
+                                       .init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "whitePSController")),
+                                       .init(image: UIImage(named: "whitePSController"))]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(.init(UINib(nibName: "ControllerCollectionViewCell",
```

Comment:
bunu her cell için yapıyosun, extension'a taşıyıp sadece `collectionView.register(type: ControllerCollectionViewCell.self)` şeklinde nası yaparsın? Hep her cell'in nibName'i class adı, identifier'ı da class adı olsun

Reply:
@aniltaskiran  cok mantikli, ekliyorum. 

`guard let cell = journeysCollectionView.dequeueReusableCell(withReuseIdentifier: "journeyCell", for: indexPath) as? JourneyCollectionViewCell else { fatalError() }`

bu kisimlar icin de withReuseIdentifier uyumlulugu icin bi extension yazayim de mi yine class alir ve class ismini kullanir, hatta discorda generic bi yapi atmistin UICollectionViewCell ler icin onu alayim.

Reply:
pardon otobus bilet uygulamasindan almisim ustteki cell kodunu 😄

`       guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "psControllerIdentifier", for: indexPath) as? ControllerCollectionViewCell else {
            fatalError()
        }`

Reply:
aynennn attigin extension baya iyi 🕺🏻

Reply:
https://github.com/trendyoliOSBootcamp/ps-controller-app-sburaksak/pull/1/commits/7e653f8d2b111be11e3139c5ee9ce1e017cad4e6

@aniltaskiran  ekledim, cok tesekkurler 🚀